cmake/CodeCoverage.cmake

```diff
@@ -0,0 +1,682 @@
+# Copyright (c) 2012 - 2017, Lars Bilke
```

Comment:
Why copy this file into our source tree rather than referencing it as a submodule the way we were before?